shared/templates/create_audit_rules_unsuccessful_file_modification_detailed.py

```diff
@@ -37,6 +37,53 @@ def generate(self, target, args):
                 },
                 "./oval/audit_rules_unsuccessful_file_modification_{0}_rule_order.xml", syscall
             )
+
+        elif target == "bash":
+            self.file_from_template(
+                "./template_BASH_audit_rules_unsuccessful_file_modification_o_creat",
+                {
+                    "SYSCALL":  syscall
+                },
+                "./bash/audit_rules_unsuccessful_file_modification_{0}_o_creat.sh", syscall
+            )
+            self.file_from_template(
+                "./template_BASH_audit_rules_unsuccessful_file_modification_o_trunc_write",
+                {
+                    "SYSCALL":  syscall
+                },
+                "./bash/audit_rules_unsuccessful_file_modification_{0}_o_trunc_write.sh", syscall
+            )
+            #self.file_from_template(
+            #    "./template_BASH_audit_rules_unsuccessful_file_modification_rule_order",
```

Comment:
We will need these as well, bu let's do that in a separate PR.

Reply:
@jan-cerny exactly.